docs/maintain-guides-how-to-create-offline-transaction.md

```diff
@@ -0,0 +1,77 @@
+---
+id: maintain-guides-how-to-create-offline-transaction
+title: Generate an offline transaction
+sidebar_label: Generate an offline transaction
+---
+
+# Generating an offline transaction
+
+There are few different ways to create an offline transaction: by using the [@polkadot@cli-signer](https://github.com/polkadot-js/tools/tree/master/packages/signer-cli), [txwrapper](https://github.com/paritytech/txwrapper), or [Subkey](https://github.com/paritytech/substrate/tree/master/bin/utils/subkey). This guide shows you how to use each one of these for different scenarios. 
+
+## polkadot@cli-signer
+
+This would be the easiest to use if you are looking for a CLI approach and have some basic understanding of public-key cryptography. You can use that to create an extrinsic and submit it as a transaction to the network directly or you sign an offline transaction and submit it over RPC or on [PolkadotJS Apps UI](https://polkadot.js.org/apps/#/toolbox).
+
+### Installation
+
+We assume you have installed [NodeJS with npm](https://nodejs.org). Run the following command:
+
+```bash
+npm install -g @polkadot/api-cli
+```
+
+### Creating a transaction
+
+Let's say we want to transfer 0.5 KSM from `DMTHrNcmA8QbqRS4rBq8LXn8ipyczFoNMb1X4cY2WD9tdBX` to `EQ64DQpzBkMcPUqUh7ddvAWmuQFnX3gCPpY8E6F9EsZjHPT`. We need to construct a payload and then use `DMTHrNcmA8QbqRS4rBq8LXn8ipyczFoNMb1X4cY2WD9tdBX` to sign it. 
+
+Executing the following command:
+
+```bash
+yarn run:signer sendOffline --account SENDER_ADDR --ws WEBSOCKET_ENDPOINT RUNTIME_MODULE.FUNCTION RECEIVER_ADDR AMOUNT
+```
+
+```bash
+yarn run:signer sendOffline --account DMTHrNcmA8QbqRS4rBq8LXn8ipyczFoNMb1X4cY2WD9tdBX --ws wss://kusama-rpc.polkadot.io/ balances.transfer EQ64DQpzBkMcPUqUh7ddvAWmuQFnX3gCPpY8E6F9EsZjHPT 500000000000
+
+```
+
+You should see 
+
+```bash
+Payload: 0x040050aab27d83bafddea5ffdb5d24fa38b98e3940d08c3744a65214cf7613930c51070088526a743501d0001a040000b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafec37cfc20a7851d7139acd706cdc47a3abc6f8b23a28c7f72098138a09e15b4df
+Signature> 
+```
+
+Then copy the payload and open another terminal to run the following command to sign it. 
```

Comment:
Just moving to another terminal does nothing.

Reply:
hm.. what do you think I 

> This kind of guide, without appropriate context, makes me nervous.
> 
> Subkey is a no-go here.

oh..so Subkey does not allow to create an offline tx? I thought it is possible. 🤔

Just moving to another terminal does nothing. <--- Are you saying that it is not quite clear why you need to do that or the result is not what you expected ?

Reply:
> oh..so Subkey does not allow to create an offline tx? I thought it is possible.

Nope

> Are you saying that it is not quite clear why you need to do that or the result is not what you expected?

Read your instructions and think about it.